src/plotting_tools.py

```diff
@@ -1,30 +1,26 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 import copy
 import os
-import time
 import matplotlib.font_manager
-import matplotlib.pyplot as plt
 
 from gi.repository import Gtk, Adw
 from matplotlib import colors
-from matplotlib.backends.backend_gtk4cairo import FigureCanvas
-from matplotlib.figure import Figure
 from matplotlib.widgets import SpanSelector
-from cycler import cycler
 
-from . import graphs, utilities, rename_label
+from . import graphs, utilities
 
 def define_highlight(self, span=None):
     """
     Create a span selector object, to highlight part of the graph.
     If a span already exists, make it visible instead
     """
+    color = self.main_window.get_style_context().lookup_color('accent_color')[1]
```

Comment:
Does this get the used accent color in the GNOME theme? That would be really nice actually, as that would ensure compatibiltiy with Gradience or perhaps Ubuntu accent colors (not sure how those work, I don't use Ubuntu). While I'm not targeting non-stock GNOME, extra compatibility is always nice. Also makes it easier when the theme slightly changes.

Reply:
I think it does, Haven't considered compatibility. My idea was just not having it hardcoded